src/addon/cc-addon-option.js

```diff
@@ -0,0 +1,139 @@
+import '../atoms/cc-img.js';
+import '../atoms/cc-toggle.js';
+import { assetUrl } from '../lib/asset-url.js';
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+
+/**
+ * A component that allows to enable or disable an add-on option.
+ *
+ * * 🎨 default CSS display: `block`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/addon/cc-addon-option.js)
+ *
+ * ## Details
+ *
+ * * If `enabled` is set to true, the option will be enabled by default. Otherwise, it will be disabled.
+ * * Emits an `input` event when the option is enabled or disabled.
+ *
+ * ## Technical details
+ * * If you want to display a warning in the option, add the `option-warning` class to an HTML element. It will be displayed
+ * with a custom color and in italic.
+ *
+ * @prop {String} logo - The logo URL of the option.
+ * @prop {String} title - Title of the option.
+ * @prop {Boolean} enabled - Enable the option by default.
+ *
+ * @event {CustomEvent<Boolean>} cc-addon-option:input - Fires when the option is enabled or disabled.
+ *
+ * @slot - The content of the option's description (text or HTML).
+ */
+export class CcAddonOption extends LitElement {
+
+  static get properties () {
+    return {
+      title: { type: String, attribute: 'title' },
+      logo: { type: String, attribute: 'logo' },
+      enabled: { type: Boolean, attribute: 'enabled' },
+    };
+  }
+
+  constructor () {
+    super();
+    this.title = null;
+    this.logo = null;
+    this.enabled = false;
+  }
+
+  _onToggleOption () {
+    this.enabled = !this.enabled;
+    dispatchCustomEvent(this, 'input', this.enabled);
+  }
+
+  render () {
+    const logo = this.logo;
+    const title = this.title;
+    const choices = [{
+      label: i18n('cc-addon-option.disabled'), value: false,
+    }, { label: i18n('cc-addon-option.enabled'), value: true }];
+
+    return html`
+      <div class="option ${classMap({ 'option--enabled': this.enabled })}">
```

Comment:
If you add `reflect:true` for `enabled` in `static get properties()`, you'll be able to style without needing this. It's not a 100% strict rule but in this case it works well I think.